File: integration-tests/mongodb-panache/src/main/java/io/quarkus/it/mongodb/panache/test/TestReactiveEntity.java
package io.quarkus.it.mongodb.panache.test;

import io.quarkus.mongodb.panache.reactive.ReactivePanacheMongoEntity;

public class TestReactiveEntity extends ReactivePanacheMongoEntity {
    public String title;
    public String category;
    public String description;
    public int cpt = 1;

    public TestReactiveEntity() {
    }

    public TestReactiveEntity(String title, String category, String description) {
        this.title = title;
        this.category = category;
        this.description = description;
    }
}

File: integration-tests/mongodb-panache-kotlin/src/main/kotlin/io/quarkus/it/mongodb/panache/test/TestReactiveEntity.kt
package io.quarkus.it.mongodb.panache.test

import io.quarkus.mongodb.panache.kotlin.reactive.ReactivePanacheMongoCompanion
import io.quarkus.mongodb.panache.kotlin.reactive.ReactivePanacheMongoEntity

class TestReactiveEntity : ReactivePanacheMongoEntity {
    companion object : ReactivePanacheMongoCompanion<TestReactiveEntity>

    lateinit var title: String
    lateinit var category: String
    lateinit var description: String
    var ctp = 1

    constructor() {}

    constructor(title: String, category: String, description: String) {
        this.title = title
        this.category = category
        this.description = description
    }
}
